core/app/[locale]/layout.tsx

```diff
@@ -117,6 +118,7 @@ export default async function RootLayout({ params, children }: Props) {
         <Notifications />
         <NextIntlClientProvider locale={locale} messages={messages}>
           <NuqsAdapter>
+            <Toaster position="top-right" />
```

Comment:
This should be provided in https://github.com/bigcommerce/catalyst/blob/soul/main/core/app/providers.tsx#L12 Do we need to add this?

Reply:
Ahh, probably what is causing my toaster issue. Good catch, I missed this in the rebase.